### PR TITLE
(fix): Add md-field for categoryDescription and fixed package.yaml(openebs)

### PR DESCRIPTION
**Pull request by aditya109**
Signed-off-by: daitya109 <adikid1996@gmail.com>

### CODE REVIEW

File: charts/generic/container-kill/container-kill.chartserviceversion.yaml [hunk 1]
```diff
@@ -2,16 +2,18 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: container-kill
-  version: 0.1.2
+  version: 0.1.1
```
Comment (ajeshbaby) [inline charts/generic/container-kill/container-kill.chartserviceversion.yaml]: Leave the version as it is. We  should not be downgrading a version.

File: charts/generic/pod-delete/pod-delete.chartserviceversion.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: pod-delete
-  version: 0.1.2
+  version: 0.1.1
```
Comment (ajeshbaby) [inline charts/generic/pod-delete/pod-delete.chartserviceversion.yaml]: Leave the version as it is. We  should not be downgrading a version.

File: charts/generic/pod-network-latency/pod-network-latency.chartserviceversion.yaml [hunk 3]
```diff
@@ -2,16 +2,18 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: pod-network-latency
-  version: 0.1.2
+  version: 0.1.1
```
Comment (ajeshbaby) [inline charts/generic/pod-network-latency/pod-network-latency.chartserviceversion.yaml]: Leave the version as it is. We  should not be downgrading a version.

File: charts/generic/pod-network-loss/pod-network-loss.chartserviceversion.yaml [hunk 4]
```diff
@@ -2,16 +2,18 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: pod-network-loss
-  version: 0.1.2
+  version: 0.1.1
```
Comment (ajeshbaby) [inline charts/generic/pod-network-loss/pod-network-loss.chartserviceversion.yaml]: Leave the version as it is. We  should not be downgrading a version.

**Review by ajeshbaby — CHANGES_REQUESTED** [review]
Added the comments for version related changes, please resolve.

File: charts/generic/container-kill/container-kill.chartserviceversion.yaml (hunk 1 above)
Comment (aditya109) [reply charts/generic/container-kill/container-kill.chartserviceversion.yaml]: fixed this

File: charts/generic/pod-delete/pod-delete.chartserviceversion.yaml (hunk 2 above)
Comment (aditya109) [reply charts/generic/pod-delete/pod-delete.chartserviceversion.yaml]: fixed this

File: charts/generic/pod-network-latency/pod-network-latency.chartserviceversion.yaml (hunk 3 above)
Comment (aditya109) [reply charts/generic/pod-network-latency/pod-network-latency.chartserviceversion.yaml]: fixed this

File: charts/generic/pod-network-loss/pod-network-loss.chartserviceversion.yaml (hunk 4 above)
Comment (aditya109) [reply charts/generic/pod-network-loss/pod-network-loss.chartserviceversion.yaml]: fixed this as well

**Review by ajeshbaby — DISMISSED** [review]
LGTM

**Review by chandankumar4 — DISMISSED** [review]
LGTM

**Comment by ksatchit** [comment]
Can you change this as well: https://github.com/litmuschaos/chaos-charts/blob/6df3d76e125f4abdaa7765fcee03db5c7d6e723d/charts/generic/generic.package.yaml#L1

**Review by ksatchit — APPROVED** [review]
LGTM

**Review by chandankumar4 — APPROVED** [review]

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Pull request merged**